test/utils.dart

```diff
@@ -168,23 +173,24 @@ Future<List<int>> fetchAsBytes(VirtualDirectory virtualDir, String path,
       virtualDir, (int port) => _fetchAsBytes(port, path, from, to));
 }
 
-Future<List> fetchContentAndResponse(VirtualDirectory virtualDir, String path,
-    {int from, int to}) async {
+Future<List> fetchContentAndResponse(VirtualDirectory? virtualDir, String path,
```

Comment:
This looks like it should probably be non-nullable?

Reply:
Done.